src/extensions/custom-view/bootstrap-table-custom-view.js

```diff
@@ -44,19 +44,19 @@ Utils.extend($.fn.bootstrapTable.defaults, {
   }
 })
 
-Utils.extend($.fn.bootstrapTable.locales, {
+Object.assign($.fn.bootstrapTable.locales, {
   formatToggleCustomViewOn () {
     return 'Show custom view'
   },
   formatToggleCustomViewOff () {
     return 'Hide custom view'
   }
 })
-Utils.extend($.fn.bootstrapTable.defaults, $.fn.bootstrapTable.locales)
+Object.assign($.fn.bootstrapTable.defaults, $.fn.bootstrapTable.locales)
 
 $.fn.bootstrapTable.methods.push('toggleCustomView')
 
-Utils.extend($.fn.bootstrapTable.Constructor.EVENTS, {
+Object.assign($.fn.bootstrapTable.Constructor.EVENTS, {
```

Comment:
Use `$.fn.bootstrapTable.events` instead of `$.fn.bootstrapTable.Constructor.EVENTS`, the same with others.
```suggestion
Object.assign($.fn.bootstrapTable.events, {
```